maestrowf/conductor.py

```diff
@@ -131,8 +132,8 @@ def monitor_study(dag, pickle_path, cancel_lock_path, sleep_time):
                  "sleep time  = %s",
                  pickle_path, cancel_lock_path, sleep_time)
 
-    study_complete = False
-    while not study_complete:
+    study_complete = StudyStatus.RUNNING
```

Comment:
I recommend renaming this variable just to be clearer.

Reply:
Yup you're right